source/ios/tools/IOSFeed/IOSFeed.cs

```diff
@@ -0,0 +1,128 @@
+                // storage here.
+                // Create the CloudBlobClient that represents the 
+                // Blob storage endpoint for the storage account.
+                CloudBlobClient cloudBlobClient = storageAccount.CreateCloudBlobClient();
+
+                CloudBlobContainer cloudBlobContainer =
+                    cloudBlobClient.GetContainerReference(Constatns.containerId);
+                await cloudBlobContainer.CreateIfNotExistsAsync();
+
+                // Set the permissions so the blobs are public.
+                BlobContainerPermissions permissions = new BlobContainerPermissions
+                {
+                    PublicAccess = BlobContainerPublicAccessType.Blob
+                };
+
+                await cloudBlobContainer.SetPermissionsAsync(permissions);
+
+                var sourceFile = Path.Combine(Constatns.frameworkPath, Constatns.frameworkName);
+                var blobGuid = Guid.NewGuid().ToString();
+                var cloudFileName = blobGuid + Constatns.frameworkName;
+
+                CloudBlockBlob cloudBlockBlob = cloudBlobContainer.GetBlockBlobReference(cloudFileName);
+
+                await cloudBlockBlob.UploadFromFileAsync(sourceFile);
+
+                BlobContinuationToken blobContinuationToken = null;
+                do
+                {
+                    var results = await cloudBlobContainer.ListBlobsSegmentedAsync(null, blobContinuationToken);
+                    // Get the value of the continuation token returned by the listing call.
+                    blobContinuationToken = results.ContinuationToken;
+                    foreach (IListBlobItem item in results.Results)
+                    {
+                        var uriString = item.Uri.ToString();
+                        if (uriString.Contains(blobGuid))
+                        {
+                            UpdatePodSpec(uriString);
+                        }
+                    }
+                } while (blobContinuationToken != null);
+            }
+            else
+            {
+                Console.WriteLine("missing valid connection string");
+            }
+        }
+
+        private static void UpdatePodSpec(string uri)
+        {
+            var sourceFile = Path.Combine(Constatns.podspecPath, Constatns.podspecName);
+            var targetFile = Path.Combine(Constatns.targetPodspecPath, Constatns.podspecName);
+            // Open the stream and read it back.
+            using (StreamReader sr = File.OpenText(sourceFile))
+            {
+                string s = "";
+                string output = "";
+                while ((s = sr.ReadLine()) != null)
+                {
+                    if (s.Length != 0)
+                    {
+                        var splits = s.Split('=');
+                        if (splits.Length > 0)
+                        {
+                            if (splits[0].Contains("spec.source"))
+                            {
```

Comment:
Are you not able to use the built in System.Uri or System.UriBuilder class? you shouldnt have to parse the tokens manually.

Reply:
i'm not parsing the uri here. i'm attaching uri to podspec file.

---
In reply to: [309843959](https://github.com/microsoft/AdaptiveCards/pull/3291#discussion_r309843959) [](ancestors = 309843959)